include/loader/shared_library.h

```diff
@@ -47,6 +48,7 @@ class SharedLibrary : public std::enable_shared_from_this<SharedLibrary> {
   Expect<void> load(const std::filesystem::path &Path) noexcept;
   Expect<void> load(const AST::AOTSection &AOTSec) noexcept;
   void unload() noexcept;
+  void invoke(std::function<void(void *Handle)> cb) { cb(Handle); }
```

Comment:
We should not use `cb`. This is unclear.

Reply:
Done. Use GetDescriptor directly from Lib->get symbol.